.github/workflows/build.yml

```
@@ -92,6 +92,11 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-features
 
+      - name: Pull latest providers
+        run: cargo run pull --branch=main
```

Not sure, but you might need the separator there

```suggestion
        run: cargo run -- pull --branch=main
```